### ISSUE TITLE
how to insert a row without a value

### ISSUE THREAD

**Issue by u84six** [hiddentao/squel#130]
I can't see a way in the insert examples to insert a row without a value. Sometimes we do this to generate a row with an incremented primary key and use that key to create a relationship between 2 tables for easy joins for tables that contain different fields that have releationships in the same table. Is there a way to insert without a value? I don't have the option to create a trigger because I need to support multiple databases.

**Comment by hiddentao** [comment]
Yep this is currently not supported out-of-the-box but will be easy to add in for the next release.

**Comment by yinrong** [comment]
hope this feature can be added soon

**Comment by yinrong** [comment]
the following code is ugly:
```
    var sql = squel.insert().into(table);
    if (Object.keys(sets).length > 0) {
      sqlSets(sql, sets);
      sql = sql.toString();
    } else {
      sql = sql.toString() + ' () VALUES ()';
    }
    that._conn.query(sql, next);
```
